dorandoran/config/settings/local.py

```diff
@@ -1,16 +1,17 @@
 from .base import *
 import os
+
 DEBUG = True
 
 ALLOWED_HOSTS = ["127.0.0.0", "localhost"]
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.mysql",
-        "NAME": os.getenv('NAME'),
-        'USER': os.getenv('USER'),
-        'PASSWORD': os.getenv('PASSWORD'),
-        'HOST': os.getenv('HOST'),
-        'PORT': os.getenv('PORT'),
+        "ENGINE": "mysql.connector.django",
+        "NAME": os.getenv("NAME"),
+        "USER": os.getenv("USER"),
+        "PASSWORD": os.getenv("PASSWORD"),
+        "HOST": os.getenv("HOST"),
+        "PORT": os.getenv("PORT"),
```

Comment:
```suggestion
        'ENGINE': 'django.db.backends.sqlite3',
        'NAME': BASE_DIR / 'db.sqlite3',
```
local에서 개발할 수 있는 환경을 제공하는 파일이기 때문에 db.sqlite3를 사용하는게 맞아요.

Reply:
local 환경에 있는 mysql 환경으로 테스트하는 것이 더 좋지 않을까요??? 실제 배포 환경에서도 aws rds의 mysql을 사용하기 때문에 더 알맞다고 생각해서 수정했었습니다

Reply:
실제 mysql 환경에서 사용하기 위해 `develop.py` 파일을 만들어 놨습니다. local.py는 외부와 연결이 차단된 상태에서 사용하기 위해 만들어 두었습니다 `ex) 기숙사`

Reply:
아 넵 이해가 가네요 수정하겠습니다